workspaces/bulk-import/plugins/bulk-import/src/components/Repositories/DeleteRepository.tsx

```diff
@@ -30,14 +30,20 @@ const DeleteRepository = ({ data }: { data: AddRepositoryData }) => {
     setOpenDialog(true);
   };
 
+  const tooltipMessage =
+    data.source === 'location'
+      ? 'Remove'
+      : 'This repository added to the app-config file. To remove it modify the file directly';
```

Comment:
typo ?
```suggestion
      : 'This repository has been added to the app-config file. To remove it modify the file directly';
```

@ShiranHi

Reply:
Wdyt about:

> This repository is part of the catalog configuration, typically in the app-config or a location entity. You must remove it from this configuration to remove it from the catalog and bulk import list.

Too long? @ciiay @debsmita1 @ShiranHi 

And btw, the item doesn't appear automatically? In the catalog it doesn, but not on the bulk import added page, or?

@dzemanov why isn't it possible to remove it? Not from the catalog, but from the bulk import added page? :)